gpm_api/tests/dataset/generate_test_dataset_assets.py

```diff
@@ -0,0 +1,280 @@
+import datetime
+import h5py
+import os
+
+from dateutil.relativedelta import relativedelta
+
+from gpm_api.configs import get_gpm_username, get_gpm_password
+from gpm_api.dataset.granule import open_granule
+from gpm_api.io import download, products
+from gpm_api.io.pps import find_pps_filepaths
+
+
+# Create asset directories #####################################################
```

Comment:
Maybe we could find a better name than asset.
`generate_test_dataset_granules.py` !